### DISCUSSION TITLE
Forage harvesters - best practice?

### DISCUSSION THREAD

**Discussion by djnekkid** [Courseplay/courseplay#7362] [General]
Hi all!

TL;DR
Im currently working on a BIG farm with more then 1000 cattle, on a season map, so my needs are rather high, currently in the 5-6 million liters per year. And thus do I need some effective way of getting both hay and silage.

I've not tested bunkers bunkers on the new version, as there are apparently some improvemens there, but they arent so easy to 'automate' anyway, so the fermenting silos have been doing their job so far.

I've mostly been baling up the grass and hay, and autopicked them from the fields, and then put them in a bale-shredder, and then use the feed-mixer. Everything can be done quite automaticly, appart from the bale shredding.

/TL:DR

So, I've been contemplating getting forage harvesters to pickup the hay and grass, but they seem to need so much baby-sitting.
So i guess the question is; Whats the 'best practice' for Forage Harvesters, especially on grass, and what tractor/truck + wagon combo works best for you?
My fields are mostly square or rectengular, with the occtional "on of the sides might have a slight angle", but no L-shapes or such.

**Comment by Tensuko** [comment]
Doesn't this work for your with grass cutting ? I know we keep drive behind the Harvester, but that shouldn't be a problem with the short pipe on most Harvester.
![cutter](https://user-images.githubusercontent.com/27344744/126062867-ad7b4658-3e78-4e44-bb0d-a091250cb1c9.png)

**Comment by djnekkid** [comment]
last test (on a test map) i used theese:
![image](https://user-images.githubusercontent.com/65364380/126065440-2033924b-0402-424a-9cc3-f1c26d33b73d.png)
![image](https://user-images.githubusercontent.com/65364380/126065446-b49830a7-8da4-44b4-babc-dce381f1da91.png)

with 
![image](https://user-images.githubusercontent.com/65364380/126065463-8aa7b391-45b4-418c-b5e4-9483a96c168b.png)
and
![image](https://user-images.githubusercontent.com/65364380/126065431-bfc02d90-73b9-4c4d-b091-db6de447669a.png)

and it were on a cut-and-winrowed course. But going from lane to lane seemed to 'but abit', and also when the wagons were filled, the tractor didn't properly 'de-attach' from the harvester.

**Comment by Tensuko** [comment]
Didn't try the Trailer yet, but it also have something to do with the fill node of the Trailers.
Maybe where is something on the way that might help you on the pull request.

**Reply by djnekkid** [reply]
any suggested trailers?

**Reply by Tensuko** [reply]
That is hard to say... I would try out Vanilla Trailers.
As I said there is maybe a better turn behaviour on the way.

**Comment by ConstructionGamer** [comment]
Sorry if its self promo, but I did do a video last year, updates have fixed problems I was having since then, but I did manage to get CP to work with forage a harvester https://youtu.be/Zue8OJP8aI0
